Failing tests

We are seeing some failing tests, that seem to be related to the identifier sequence stored procedure:

```“[ERROR] Failures: “, “[ERROR]  DatasetsIT.testLinkingDatasets:1981 Expected status code <201> doesn’t match actual status code <500>.“, “”, “[ERROR] Errors: “, “[ERROR]  DatasetsIT.testCreateDataset:132 » IllegalArgument Cannot get property ‘id’ on...“, “[ERROR]  DatasetsIT.testFileChecksum:1223 » IllegalArgument Cannot get property ‘id’ on...“, “[ERROR]  DatasetsIT.testPrivateUrl:805 » IllegalArgument Cannot get property ‘id’ on nu...“, “[ERROR]  DatasetsIT.testSequentialNumberAsIdentifierGenerationStyle:749 » IllegalArgument”, “[INFO] “, “[ERROR] Tests run: 130, Failures: 1, Errors: 4, Skipped: 4”, “[INFO] ”```

--- a/src/main/java/edu/harvard/iq/dataverse/Dataset.java
+++ b/src/main/java/edu/harvard/iq/dataverse/Dataset.java
@@ -74,7 +74,7 @@ import edu.harvard.iq.dataverse.util.StringUtil;
         name = "Dataset.generateIdentifierAsSequentialNumber",
         procedureName = "generateIdentifierAsSequentialNumber",
         parameters = {
-            @StoredProcedureParameter(mode = ParameterMode.OUT, type = Integer.class, name = "identifier")
+            @StoredProcedureParameter(mode = ParameterMode.OUT, type = Integer.class)
         }
 )
 @Entity

